Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -301,8 +302,10 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertFalse(
             scope.process(command: RUMStopViewCommand.mockWith(identity: mockView))
         )
-        let event = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMView>.self).last)
-        XCTAssertEqual(event.model.view.action.count, 1, "View should record 1 action")
+        let viewEvent = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMView>.self).last)
+        XCTAssertEqual(viewEvent.model.view.action.count, 1, "View should record 1 action")
+        let actionEvent = try XCTUnwrap(output.recordedEvents(ofType: RUMEvent<RUMAction>.self).last)
+        XCTAssertEqual(actionEvent.model.action.target?.name, actionName, "ViewScope should pass action properties")
```

Comment:
`RUMEvent<RUMAction>` assertion is not important in `RUMViewScope` **unit** tests. In this suite we only assert `RUMEvent<RUMView>` things. 

The way `RUM` tests are structured is:
* each `XScope` tests only its minimal logic (the unit),
* integration logic is tested in `RUMMonitorTests`.

Having this convention keeps each scope tests minimal and does not duplicate child scope assertions (no need to refactor parent scope tests due to implementation detail change in the child scope).